driver/esp32/espidf.c

```diff
@@ -9,6 +9,7 @@
 #include "freertos/task.h"
 #include "esp_system.h"
 #include "soc/cpu.h"
+#include "lvgl/src/draw/sw/lv_draw_sw.h"
```

Comment:
Is this header really needed?

Reply:
Yes, because we call `lv_draw_sw_rgb565_swap()` function in `ili9xxx_flush()`

Reply:
Sorry, not needed, `lvgl.h` is already included